issue-wanted.cabal

```diff
@@ -121,6 +121,21 @@ test-suite issue-wanted-test
   import:              common-options
   type:                exitcode-stdio-1.0
   hs-source-dirs:      test
+
   main-is:             Spec.hs
+  other-modules:       Test.Assert
+                       Test.Common
+                       Test.Core.Issue
+                       Test.Core.Repo
+                       Test.Db
+
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
   build-depends:       issue-wanted
+                     , hedgehog
+                     , hspec
+                     , mtl
+                     , hedgehog ^>= 0.6.1
```

Comment:
`hedgehog` is specified twice in the `build-depends` list. Also, let's use `1.0` version.